lib/assessment/rubric_tester.py

```diff
@@ -180,6 +181,14 @@ def get_s3_folder(s3, path_from_s3_root):
             continue
         bucket.download_file(obj.key, target)
 
+# Retrieve params from private github repository and store in experiments. Requires ssh key
+def get_params_github(branch=None):
+  git_repo = "git@github.com:code-dot-org/aitt_release_data.git"
+  if branch:
+    subprocess.run(["git", "clone", git_repo, "-b", branch, f"./{branch}"], capture_output=True, cwd='./experiments')
```

Comment:
afaik, there is no way to test against a local branch without first pushing a branch to origin